client/analysis/output.go

```diff
@@ -8,44 +8,54 @@ import (
 	"encoding/json"
 	"fmt"
 	"os"
+	"strconv"
 	"strings"
 
 	"github.com/fatih/color"
 	"github.com/globocom/huskyCI/client/types"
 	"github.com/globocom/huskyCI/client/util"
 )
 
+var outputJSON types.JSONOutput
+var goResults types.GoResults
+var pythonResults types.PythonResults
+var javaScriptResults types.JavaScriptResults
+var rubyResults types.RubyResults
+
 // CheckMongoDBContainerOutput will validate the output of a given container.
 func CheckMongoDBContainerOutput(container types.Container) {
 
 	switch container.SecurityTest.Name {
 	case "enry":
 	case "gosec":
-		PrintGosecOutput(container.COutput, container.CInfo)
+		PrepareGosecOutput(container.COutput, container.CInfo)
+		outputJSON.GoResults = goResults
 	case "bandit":
-		PrintBanditOutput(container.COutput, container.CInfo)
+		PrepareBanditOutput(container.COutput, container.CInfo)
+		outputJSON.PythonResults.BanditOutput = pythonResults.BanditOutput
 	case "retirejs":
-		PrintRetirejsOutput(container.COutput, container.CInfo)
+		PrepareRetirejsOutput(container.COutput, container.CInfo)
+		outputJSON.JavaScriptResults.RetirejsResult = javaScriptResults.RetirejsResult
 	case "brakeman":
-		PrintBrakemanOutput(container.COutput, container.CInfo)
+		PrepareBrakemanOutput(container.COutput, container.CInfo)
+		outputJSON.RubyResults.BrakemanOutput = rubyResults.BrakemanOutput
 	case "safety":
-		PrintSafetyOutput(container.COutput, container.CInfo)
+		PrepareSafetyOutput(container.COutput, container.CInfo)
+		outputJSON.PythonResults.SafetyOutput = pythonResults.SafetyOutput
 	default:
 		fmt.Println("[HUSKYCI][ERROR] securityTest name not recognized:", container.SecurityTest.Name)
 		os.Exit(1)
 	}
 }
 
-// PrintGosecOutput will print the Gosec output.
-func PrintGosecOutput(mongoDBcontainerOutput string, mongoDBcontainerInfo string) {
+// PrepareGosecOutput will prepare Gosec output.
+func PrepareGosecOutput(mongoDBcontainerOutput string, mongoDBcontainerInfo string) {
 
 	if mongoDBcontainerInfo == "No issues found." {
-		color.Green("[HUSKYCI][*] Gosec :)\n\n")
 		return
 	}
 
 	foundVuln := false
```

Comment:
This old `foundVuln` logic must be reviewed. Checking this new code, we can see that this variable will always be false.